packages/shopping/public/api.js

```diff
@@ -3,6 +3,33 @@
 'use strict';
 
 const api = {
+
+  async getUser(userId) {
+    const token = localStorage.getItem('shoppyToken');
+    const url = apiUrl + `/users/${userId}`;
+    return $.ajax({
+      type: 'GET',
+      url: url,
+      headers: {
+        Authorization: `Bearer ${token}`,
+      }
+    }).promise();
+  },
+
+  async updateUser(userId, body) {
+    const token = localStorage.getItem('shoppyToken');
+    const url = apiUrl + `/users/${userId}`;
+    return $.ajax({
+      type: 'POST',
+      url: url,
+      data: JSON.stringify(body),
```

Comment:
why do we need to stringify body here? isn't $.ajax taking in a JSON object anyways? why would obj.data need to be stringified then? Just curious.

Reply:
https://api.jquery.com/jquery.ajax/ says data can be PlainObject or string.

Reply:
For some reason it's not working. Let's refactor some other time.